### PR TITLE
Let's add a poem!

### PR DESCRIPTION
**Pull request by alexsoble**
Hey team! can you suggest a good ending to this poem?

### CODE REVIEW

File: CHECKLIST.MD [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Git commands!
 
+Note: check permissions to the playground first!
```
Comment (karareinsel) [inline CHECKLIST.MD:3]: This is a test.

File: CHECKLIST.MD [hunk 2]
```diff
@@ -1,5 +1,7 @@
...
+
```
Comment (sawtoothwave) [inline CHECKLIST.MD:4]: this seems like a great idea!

File: poem.md [hunk 3]
```diff
@@ -0,0 +1,2 @@
+Roses are red!
```
Comment (sawtoothwave) [inline poem.md:1]: this seems like a very good start for a poem

File: CHECKLIST.MD (hunk 2 above)
Comment (karareinsel) [reply CHECKLIST.MD:4]: awesome

**Review by karareinsel — APPROVED** [review]

**Pull request merged**